DailyKit/CallManager/Models/CallCamera.swift

```diff
@@ -0,0 +1,41 @@
+import Daily
+import AVFoundation
+
+public struct CallCamera: Equatable {
+    public enum FacingMode: Equatable {
+        case user, environment
+    }
+
+    public let video: CallMuteState
+    public let mode: FacingMode
+
+    public init(video: CallMuteState, mode: FacingMode) {
+        self.video = video
+        self.mode = mode
+    }
+
+    public var isMuted: Bool { video == .muted }
```

Comment:
+1!

It's an unofficial convention to put properties above initializers in Swift.

Reply:
Properties above initializers isn't something I've seen be a hard style preference in the work I've done, but it is common. Personally I prefer more flexibility to order members within a given type. I think in many cases it can make initial reading and maintenance easier to have a property grouped next to the methods that depend on it, or to have some related members grouped together instead of spread around a file based on member type or visibility. 

Here's one example of the ["logical ordering"](https://google.github.io/swift/#type-variable-and-function-declarations) perspective.

> The order of types, variables, and functions in a source file, and the order of the members of those types, can have a great effect on readability. However, there is no single correct recipe for how to do it; different files and different types may order their contents in different ways.
>
> What is important is that each file and type uses some logical order, which its maintainer could explain if asked. For example, new methods are not just habitually added to the end of the type, as that would yield “chronological by date added” ordering, which is not a logical ordering.

In this case I think properties above the initializer is more logical because these are simple state types. I'll move it up here and in the other `Call*` structs. Thanks both of you for mentioning that.

Reply:
Afaict it's by far the most prevalent unspoken community convention.

Even Apple uses it pretty consistently:
https://docs.swift.org/swift-book/documentation/the-swift-programming-language/initialization/